### ISSUE TITLE
(woad '~') crashes

### ISSUE THREAD

**Issue by ohAitch** [urbit/urbit#327]
, `woad` should instead return `(unit ,@t)` that is null for invalidly cord-formatted spans, and be used with sear. Or flat-out be a parsing rule.

**Comment by juped** [comment]
A parsing rule is definitely the way to go - the whole section could benefit from it, but the stuff that works can stay untouched for now.

**Comment by ohAitch** [comment]
I believe standard protocol is to let the "fixes #foo" auto-close it once the relevant commit is merged into master; nice work however :3
